errors/errors.go

```diff
@@ -9,7 +9,7 @@ type QueryError struct {
 	Locations     []Location    `json:"locations,omitempty"`
 	Path          []interface{} `json:"path,omitempty"`
 	Rule          string        `json:"-"`
-	ResolverError error         `json:"-"`
+	ResolverError error         `json:"extensions,omitempty"`
```

Comment:
Shouldn't we name this field `Extensions` with type `map[string]interface{}`. This way it can still contain an error as well as other fields. For example, it might be a validation error and you might include the error(s) as well as the invalid input field(s) or you might want to include request ID, trace ID or other fields that are not directly related to the error 🤔

Reply:
I agree with @pavelnikolov  that the name should be `Extensions` and the type should be a `map[string]interface{}`